sdk/java/src/main/java/com/chain/api/CoreConfig.java

```diff
@@ -94,53 +162,103 @@
     }
   }
 
+  /**
+   * Get info on specified Chain Core.
```

Comment:
I think we use the third-person present in Java comments, so "Gets" instead of "Get". Ditto elsewhere.